### PR TITLE
Finalize release

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

File: packages/apps/desktop-viewer-test/src/frontend/components/home/Home.scss
Comment (ben-polinsky) [inline packages/apps/desktop-viewer-test/src/frontend/components/home/Home.scss:1]: How are css variables pulled in from itwinui?

File: packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/BriefcaseStatus.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 *--------------------------------------------------------------------------------------------*/
 @import '@itwin/itwinui-variables/index.css';
```
Comment (ben-polinsky) [inline packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/BriefcaseStatus.scss:5]: Why does this import remain but in other scss files it is removed?

File: packages/apps/desktop-viewer-test/src/frontend/components/home/Home.scss
Comment (aruniverse) [reply packages/apps/desktop-viewer-test/src/frontend/components/home/Home.scss:1]: theyre pulled in via the global css file being imported at the root

File: packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/BriefcaseStatus.scss (hunk 1 above)
Comment (aruniverse) [reply packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/BriefcaseStatus.scss]: thank you, nice catch

**Pull request merged**